about.html

```diff
@@ -0,0 +1,106 @@
+      <p class="headline-contact">Please contact us per Email for any further questions about La Rienda World Tour!<br><a href="#" class="headline-email">worldtour2023@larienda.com</a></p>
+    </section>
+    <section class="presentation-container">
+      <h2 class="speakers-title">History</h2>
+      <hr class="title-separator">
+      <div class="presentation-content">
+        <p class="presentation-text">La Rienda is a band born in the city of Mendoza, Argentina in 2009. It's a group of young musicians who propose a particular style having traditional dances and Argentine folk music as a common language.<br> This group has toured numerous stages throughout Argentina, winners of the Diario UNO Stage Awards for the best traditional folk group, recognized by the Honorable Chamber of Senators of the province of Mendoza for their contribution to regional culture.<br>In his repertoire sounds and melodies are combined with the authenticity of modern instruments</p>
+        <div class="presentation-image-container"><img src="./images/logo.png" alt="Logo"></div>
+      </div>
+    </section>
+    <hr class="section-separator">
+    <section class="history-container">
+      <h2 class="speakers-title">Shows</h2>
+      <hr class="title-separator">
+      <p class="history-text">Take a look at the last two shows wich took place in Chile and in Uruguay</p>
+      <ul class="history-list">
+        <li class="history-card">
+          <a href="#" class="history-card-content">
+            <div class="overlay"></div>
+            <img class="history-card-image" src="./images/Santiago.jpg" alt="image">
+            <h3 class="history-card-title">2020</h3>
+            <p class="history-card-text">Santiago, Chile</p>
+          </a>
+        </li>
+        <li class="history-card">
+          <a href="#" class="history-card-content">
+            <div class="overlay"></div>
+            <img class="history-card-image" src="./images/Montevideo.jpg" alt="image">
+            <h3 class="history-card-title">2021</h3>
+            <p class="history-card-text">Montevideo, Uruguay</p>
+          </a>
+        </li>
+      </ul>
+    </section>
+    <section class="partners-container lg-none">
+      <h2 class="partners-title">Partners</h2>
+      <hr class="title-separator">
+      <ul class="partners-list">
+        <li class="partner-card"><a href="#"><img class="partner-image" src="./images/Spotify.png" alt="partner logo"></a></li>
+        <li class="partner-card"><a href="#"><img class="partner-image" src="./images/instagram-white.png" alt="partner logo"></a></li>
+        <li class="partner-card"><a href="#"><img class="partner-image" src="./images/amazon-music.png" alt="partner logo"></a></li>
+      </ul>
+    </section>
+  </main>
+  <footer class="footer-container lg-dark">
```

Comment:
- Optional
The footer of your application should be personalized as per your project. Currently, the logo and description in your footer are from the original design, when it should be the logo of your application along with its details. It would make much more sense that way. Also, since you will be putting this project on your portfolio, it is super important to keep things relevant to your project.